crates/ide-assists/src/utils.rs

```diff
@@ -334,7 +334,7 @@ fn calc_depth(pat: &ast::Pat, depth: usize) -> usize {
 pub(crate) fn find_struct_impl(
     ctx: &AssistContext<'_>,
     adt: &ast::Adt,
-    name: &str,
+    names: Vec<String>,
```

Comment:
```suggestion
    names: &[String],
```
We don't need ownership here, so `&[...]` is more general

Reply:
Nice catch!